urllib3/util/ssl_.py

```diff
@@ -110,7 +110,7 @@ def set_ciphers(self, cipher_suite):
                 )
             self.ciphers = cipher_suite
 
-        def wrap_socket(self, socket, server_hostname=None):
+        def wrap_socket(self, socket, server_hostname=None, server_side=False):
```

Comment:
Yeah, I stole this name from the actual `wrap_socket` function in the `ssl` module. So you can take it up with python-dev. ;)